src/com/oltpbenchmark/benchmarks/wikipedia/WikipediaLoader.java

```diff
@@ -477,15 +491,20 @@ private void loadRevision() throws SQLException {
                 
                 // Insert the text
                 int col = 1;
-                textInsert.setInt(col++, rev_id); // old_id
+                if (this.getDatabaseType() != DatabaseType.CUBRID) {
```

Comment:
Why do you need this?

Reply:
Old_id is an auto_increment field and we skip setting this field here.